metalearners/cross_fit_estimator.py

```diff
@@ -337,8 +338,28 @@ def predict_proba(
             oos_method=oos_method,
         )
 
-    def score(self, X, y, sample_weight=None, **kwargs):
-        raise NotImplementedError()
+    def score(
+        self,
+        X: Matrix,
+        y: Vector,
+        is_oos: bool,
+        oos_method: OosMethod | None = None,
+        sample_weight: Vector | None = None,
+    ) -> float:
+        """Return the coefficient of determination of the prediction if the estimator is
+        a regressor or the mean accuracy if it is a classifier."""
+        if self._estimator_type == "classifier":
```

Comment:
Do you happen to know what the trade-off between doing this and using `sklearn`'s `is_classifier` is here?

E.g. https://github.com/Quantco/metalearners/blob/9e00603d5b16d03981294a9d3c7f74393d41ad13/metalearners/cross_fit_estimator.py#L165

Reply:
None, I just didn't think about it 😅 . I think it's better to use the `sklearn` methods.
[108f4f3](https://github.com/Quantco/metalearners/pull/63/commits/108f4f381b5273635995b6f68d37d33244276cc9)